src/components/sections/GallerySection.tsx

```diff
@@ -1,25 +1,20 @@
-import { galleryImages } from '../gallery/AppData';
+import { galleryPhotos } from '../../AppData';
+import { ImgContainer } from '../gallery/ImgContainer';
+import { Section } from '../shared/Section';
 
 export const GallerySection = () => {
   return (
-    <section id="galerie" className="w-full py-16 sm:py-24">
-      <div className="w-full px-4 mx-auto max-w-screen-desktop desktop:px-0">
-        <h2 className="text-2xl sm:text-3xl font-semibold uppercase font-numbers mb-10 text-center">
+    <Section id="gallery" className="py-16 sm:py-24">
```

Comment:
Ff you want the id value to be set as "gallery", then change it also in NavItems.ts
because the sidebar <a> will not work